meta-linaro-mbl/meta-optee/recipes-security/optee/optee-os_%.bbappend

```diff
@@ -12,16 +12,17 @@ MBL_OPTEE_VERSION = "3.4.0"
 MBL_OPTEE_VERSION_imx8mmevk-mbl = "3.2.0"
```

Comment:
I suppose all platforms now are in the `3.4.0`. If so, we need to remove the `MBL_OPTEE_VERSION_imx8mmevk-mbl = "3.2.0"` line.

Reply:
ok. removed. Thanks.